src/main/java/core/basesyntax/FigureListToDo.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+
+public interface FigureListToDo {
+
+    String getName();
+
+    double area();
+}
```

Comment:
Sorry, my fault
in previous comment I meant that we need an interface only for `area` method
`getName` can be inside `Figure` class 

also, let's try to find a better name for this interface (after moving `getName` to the `Figure` class)